manim/mobject/boolean_ops.py

```diff
@@ -0,0 +1,291 @@
+
+    def _convert_skia_path_to_vmobject(self, path: SkiaPath) -> VMobject:
+        """Converts SkiaPath back to VMobject.
+        Parameters
+        ==========
+        path:
+            The SkiaPath to convert.
+
+        Returns
+        =======
+        VMobject:
+            The converted VMobject.
+        """
+        vmobject = self
+        current_path_start = np.array([0, 0, 0])
+
+        for path_verb, points in path:
+            if path_verb == PathVerb.MOVE:
+                parts = self._convert_2d_to_3d_array(points)
+                for part in parts:
+                    current_path_start = part
+                    vmobject.start_new_path(part)
+                    # vmobject.move_to(*part)
+            elif path_verb == PathVerb.CUBIC:
+                n1, n2, n3 = self._convert_2d_to_3d_array(points)
+                vmobject.add_cubic_bezier_curve_to(n1, n2, n3)
+            elif path_verb == PathVerb.LINE:
+                parts = self._convert_2d_to_3d_array(points)
+                vmobject.add_line_to(parts[0])
+            elif path_verb == PathVerb.CLOSE:
+                if config.renderer == "opengl":
+                    vmobject.close_path()
+                else:
+                    vmobject.add_line_to(current_path_start)
+            elif path_verb == PathVerb.QUAD:
+                n1, n2 = self._convert_2d_to_3d_array(points)
+                vmobject.add_quadratic_bezier_curve_to(n1, n2)
+            else:
+                raise Exception("Unsupported: %s" % path_verb)
+        return vmobject
+
+
+class Union(_BooleanOps):
+    """Union of two or more :class:`~.VMobject`s. This returns the common region of
+    the :class:`~VMobject`s.
+
+    Parameters
+    ==========
+    vmobjects
+        The :class:`~.VMobject` to find the union.
+
+    Raises
+    ======
+    ValueError
+        If less the 2 :class:`~.VMobject` are passed.
+
+    Example
+    =======
+
+    .. manim:: UnionExample
+        :save_last_frame:
+
+        class UnionExample(Scene):
+            def construct(self):
+                sq = Square(color=RED, fill_opacity=1)
+                sq.move_to([-2, 0, 0])
+                cr = Circle(color=BLUE, fill_opacity=1)
+                cr.move_to([-1.3, 0.7, 0])
+                un = Union(sq, cr, color=GREEN, fill_opacity=1)
+                un.move_to([1.5, 0.3, 0])
+                self.add(sq, cr, un)
+
+    """
+
+    def __init__(self, *vmobjects: VMobject, **kwargs) -> None:
+        if len(vmobjects) < 2:
+            raise ValueError("At least 2 mobjects needed for Union.")
+        super().__init__(**kwargs)
+        paths = []
+        for vmobject in vmobjects:
+            paths.append(self._convert_vmobject_to_skia_path(vmobject))
+        outpen = SkiaPath()
+        union(paths, outpen.getPen())
+        self._convert_skia_path_to_vmobject(outpen)
+
+
+class Difference(_BooleanOps):
+    """Subracts one :class:`~.VMobject` from another one.
+    Parameters
```

Comment:
Prevent these:
![image](https://user-images.githubusercontent.com/83535735/134204603-398c9a94-0bff-4eae-afdf-aaedd8d04b69.png)


```suggestion
    """Subracts one :class:`~.VMobject` from another one.
    
    Parameters
```